Formula/nginx.rb

```diff
@@ -18,6 +18,14 @@ class Nginx < Formula
   depends_on "pcre"
 
   def install
+    # keep clean copy of source for compiling dynamic modules e.g. passenger
+    (share/"src").mkpath
+    mkdir "src_rebuild" do
+      system "tar", "-xf", cached_download, "--strip-components", "1", "--exclude='conf contrib LICENSE README html man CHANGES.ru CHANGES'"
```

Comment:
How much of a difference in size do the `excludes` make? I think it's probably worth only excluding stuff that's making a meaningful difference in size e.g. images just to avoid pain in future if stuff in the tarball moves around.

Also, at this point the `buildpath` already contains a complete copy of the source so it's probably just worth compressing that and passing the `--exclude` there instead rather than uncompressing the same data twice.

Reply:
The excludes make very little difference to the size of the resulting `tar.xz`, there are no images.